python/vote-text-bot/bot.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''
+Hangouts Chat simple text-only vote bot in Python App Engine
+
+@see developers.google.com/hangouts/chat/how-tos/cards-onclick
+'''
+
+import json
+import webapp2
+
+def createMessage(voter, vote_count=0, update=False):
```

Comment:
Please follow [PEP 8 naming](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables): `create_message`.

Reply:
DONE